src/WebJobs.Script/OutOfProc/Http/MessageExtensions/HttpMessageConversionExtensions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.WebJobs.Script.OopHttp
+{
+    internal static class HttpMessageConversionExtensions
+    {
+        private static readonly JsonSerializerSettings _datetimeSerializerSettings = new JsonSerializerSettings { DateParseHandling = DateParseHandling.None };
+
+        public static Task<object> ToObject(this HttpResponseMessage responseMessage)
+        {
+            return HttpMessageExtensionUtilities.ConvertFromHttpMessageToExpando(responseMessage);
+        }
+
+        public static string ToHttpHeader(this object value)
+        {
+            if (value == null)
+            {
+                return string.Empty;
+            }
+            if (value is JObject jobj)
+            {
+               return jobj.ToString();
+            }
+            else if (value is string str)
+            {
+                return str;
+            }
+            else if (value is HttpRequest request)
+            {
+                return GetStringRepresentationOfBody(request);
+            }
+
+            return value.ToString();
+        }
+
+        private static string GetStringRepresentationOfBody(HttpRequest request)
```

Comment:
Was the intention to have this be an extension method? If so, we should move to `HttpRequestExtensions`

Reply:
do you recommend static methods instead of extensions? the purpose of these helpers is to convert from input binding data to Http Request post body